events/set_instance_tags/create.json

```diff
@@ -1,12 +1,13 @@
 {
-  "RequestType" : "Create",
-  "RequestId" : "unique_id",
-  "ResponseURL" : "https://cloudformation-custom-resource-response-useast1.s3.amazonaws.com/arn:aws:cloudformation:us-east-1:237179673806:stack/cfn-cr-synapse-tagger/",
-  "ResourceType" : "Custom::EC2InstanceTagger",
-  "LogicalResourceId" : "EC2InstanceTagger",
-  "StackId" : "arn:aws:cloudformation:us-east-1:237179673806:stack/SC-237179673806-pp-h2q3pe4itf5sk/14a082d0-7e92-11ea-a356-120fbf0f0187",
-  "ServiceToken": "arn:aws:lambda:us-east-1:237179673806:function:set-synapse-tags-macro-SetInstanceTagsFunction-4PNZHTTFOFA3",
-  "ResourceProperties" : {
-    "InstanceId" : "i-06c3d8ac9b7953f43"
+  "RequestType":"Create",
+  "ServiceToken":"arn:aws:lambda:us-east-1:465877038949:function:khai-cfn-cr-synapse-tagger-SetInstanceTagsFunction-P2RGCZEI3OW6",
+  "ResponseURL":"https://cloudformation-custom-resource-response-useast1.s3.amazonaws.com/arn%3Aaws%3Acloudformation%3Aus-east-1%3A465877038949%3Astack/SC-465877038949-pp-qv7cvlggt75lo/1a69d180-fe76-11ea-9dd7-0ad52a854211%7CTagInstance%7Ccd91ab0e-88e6-4cfd-9a03-322420db1dd9?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20200924T150008Z&X-Amz-SignedHeaders=host&X-Amz-Expires=7200&X-Amz-Credential=AKIA6L7Q4OWTVZGMUJ7R%2F20200924%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=f08e5c8bb62340f0104253b460b0d4333d2af80d624b5accaf2dfdb464693a5b",
+  "StackId":"arn:aws:cloudformation:us-east-1:465877038949:stack/SC-465877038949-pp-qv7cvlggt75lo/1a69d180-fe76-11ea-9dd7-0ad52a854211",
+  "RequestId":"cd91ab0e-88e6-4cfd-9a03-322420db1dd9",
+  "LogicalResourceId":"TagInstance",
+  "ResourceType":"Custom::SynapseTagger",
+  "ResourceProperties":{
+    "ServiceToken":"arn:aws:lambda:us-east-1:465877038949:function:khai-cfn-cr-synapse-tagger-SetInstanceTagsFunction-P2RGCZEI3OW6",
```

Comment:
Why is the `ServiceToken` in both the out object and this inner object? I think it is only necessary in `ResourceProperties`.

Reply:
good question but  i have no idea.  This is all done by AWS.  i just got this event from the logs, when the handler is called here https://github.com/Sage-Bionetworks-IT/cfn-cr-synapse-tagger/blob/master/set_tags/set_instance_tags.py#L43